contracts/TickerRegistry.sol

```diff
@@ -86,6 +100,14 @@ contract TickerRegistry is ITickerRegistry, Ownable, Util {
         return true;
     }
 
+    /**
+     * @dev set the ticker registration fee in POLY tokens
+     * @param _registrationFee registration fee in POLY tokens (base 18 decimals)
+     */
+    function setPolyRegistrationFee(uint256 _registrationFee) public onlyOwner {
```

Comment:
change the function name as we are changing the registration fee, not setting the fee for one time
it should be named as changePolyRegistrationFee and also add the event for logging the same. with old value and new value as the arguments of the event.